tracker/tracker.go

```diff
@@ -478,18 +481,21 @@ func (t *Tracker) processNextItem(data DeployMessage) (success bool) {
 	case "http":
 		switch messageType := data.Type; messageType {
 		case "deployFinished":
-			success = t.sendDeploymentEventHTTP(fmt.Sprintf("%s/finished", t.deployStatusEndpointAPI), data)
+			success = t.sendDeploymentEventHTTP(fmt.Sprintf("%s/finished", t.KubeDeployEndpointAPI), data)
 		case "deployStatus":
-			success = t.sendDeploymentEventHTTP(t.deployStatusEndpointAPI, data)
+			success = t.sendDeploymentEventHTTP(t.KubeDeployEndpointAPI, data)
 		case "deployFailedLogs":
-			success = t.sendDeploymentEventHTTP(fmt.Sprintf("%s/podlogs", t.deployStatusEndpointAPI), data)
+			success = t.sendDeploymentEventHTTP(fmt.Sprintf("%s/podlogs", t.KubeDeployEndpointAPI), data)
 		default:
 			log.WithFields(log.Fields{"message_type": messageType}).Warn("Unknown message type for http endpoint")
 			success = true //Prevent Retry for bad message
 		}
 	//Send to sqs
 	case "sqs":
-		success = t.sendDeploymentEventSQS(t.deployStatusEndpointAPI, data)
+		// Send to kube-deploy sqs
+		success = t.sendDeploymentEventSQS(t.KubeDeployEndpointAPI, data)
+		// Send to robbie sqs
+		success = t.sendDeploymentEventSQS(t.RobbieEndpointAPI, data)
```

Comment:
this basically breaks success usage.. what if first success failed but second success succeeded?

Reply:
Will revise the logic to be more reasonable